app/controllers/api/v1/admin/mentors_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Api::V1::Admin::MentorsController < ApplicationController
```

Comment:
Good start on the controller namespacing

Reply:
Thank you.